doc/nrf/releases_and_maturity/migration/migration_bicr_nRF54H20.rst

```diff
@@ -0,0 +1,478 @@
+
+.. note::
+   Configuration values use hexadecimal format, even if the ``0x`` prefix is not shown in DTS syntax.
+
+In the JSON configuration, GPIO port assignments are specified within the ``ioPortNumber`` object, with each mode explicitly indicated.
+
+The available port configuration keys include:
+
++------+--------------------+
+| Port | Port configuration |
+|      | key                |
++======+====================+
+| P1   | p1Supply           |
+| P2   | p2Supply           |
+| P6   | p6Supply           |
+| P7   | p7Supply           |
+| P9   | p9Supply           |
++------+--------------------+
+
+The supported operating modes for these ports are:
+
++--------------------------+--------------------+
+| Supported Operating Mode | JSON configuration |
++==========================+====================+
+| Disconnected             | DISCONNECTED       |
+| Shorted                  | SHORTED            |
+| External1v8              | EXTERNAL_1V8       |
+| ExternalFull             | EXTERNAL_FULL      |
++--------------------------+--------------------+
+
+Based on the DTS example, the corresponding JSON values are the following:
+
++----------------------+----------------------+----------------------+
+| GPIO ports from the  | DTS configuration    | JSON configuration   |
+| example              |                      |                      |
++======================+======================+======================+
+| P1                   | ``<gpio1 2>``        | EXTERNAL_1V8         |
+| P2                   | ``<gpio2 2>``        | EXTERNAL_1V8         |
+| P6                   | ``<gpio6 2>``        | EXTERNAL_1V8         |
+| P7                   | ``<gpio7 2>``        | EXTERNAL_1V8         |
+| P9                   | ``<gpio9 4>``        | EXTERNAL_FULL        |
++----------------------+----------------------+----------------------+
+
+The resulting JSON configuration is structured as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortPower": {
+       "p1Supply": "EXTERNAL_1V8",
+       "p2Supply": "EXTERNAL_1V8",
+       "p6Supply": "EXTERNAL_1V8",
+       "p7Supply": "EXTERNAL_1V8",
+       "p9Supply": "EXTERNAL_FULL"
+     }
+   }
+
+GPIO power drive and impedance configuration
+============================================
+
+This DTS option configures the ``BICR`` → ``IOPORT.DRIVECTRL0`` register:
+
+.. code-block:: dts
+
+   ioport-drivectrls = <&gpio6 50>, <&gpio7 50>;
+
+This section specifies the IO port impedance settings for **P6** and **P7**.
+
+As with the GPIO power configuration, each GPIO reference indicates the corresponding port number, while the second value indicates the port's impedance in Ohms.
+
+In this example, both **P6** and **P7** are configured to use a 50 Ohm impedance.
+
+The nRF54H20 SoC allows you to select from these supported impedance values:
+
+* 33 Ohms
+* 40 Ohms
+* 50 Ohms
+* 66 Ohms
+* 100 Ohms
+
+For more information, consult the datasheet.
```

Comment:
```suggestion
For more information, refer to the datasheet.
```
Link to the datasheet?

Reply:
Not sharable at the moment. Removing the sentence